.github/workflows/cd.yml

```diff
@@ -0,0 +1,66 @@
+---
+name: Publish
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: '0 5 * * 1'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+
+  build:
+    name: Build Jupyter Notebooks and HTML
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup GitHub Pages
+        uses: actions/configure-pages@v5
+
+      - name: Setup pixi
+        uses: prefix-dev/setup-pixi@19eac09b398e3d0c747adc7921926a6d802df4da  # v0.8.8
+        with:
+          cache: true
+          cache-write: true
+
+      - name: Build executed notebooks and HTML
+        run: pixi run build
+
+      - name: Upload executed notebooks as GitHub artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: executed-notebooks
+          path: _build/jupyter_execute/tutorials
+          if-no-files-found: error
```

Comment:
Do we need to store the .ipynb outputs for anything?

Reply:
They can be useful for debugging. They are [retained for 90 days](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/storing-and-sharing-data-from-a-workflow).

Reply:
I did not intentionally keep this in, but now that I look at it, I think keeping the `ipynb` as an internal debugging artifact is harmless and could be occasionally helpful.

Reply:
...But I'm not attached. We could strip it out and wait to re-instate until/unless a need arises.